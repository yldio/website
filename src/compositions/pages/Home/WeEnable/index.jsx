import React, { Fragment } from 'react';
import { Grid, Row, Col } from 'react-styled-flexboxgrid';
import styled from 'styled-components';

import List, { Item } from 'components/CheckList';
import PageSection from 'components/PageSection';
import { H2 } from 'components/Typography';

const CheckList = styled(List)`
  margin-top: 60px;
`;

const WeEnable = () => (
  <Fragment>
    <PageSection data-selector="home:community">
      <Grid data-selector="home:community">
        <Row>
          <Col xs={12}>
            <H2 decorated>What do we do</H2>
          </Col>
        </Row>
        <Row>
          <Col xs={12}>
            <CheckList>
              <Item>Transform, innovate and differentiate at speed</Item>
              <Item>
                Grow capability inside your organisation - consulting, training,
                pairing and education
              </Item>
              <Item>
                Adopt cutting edge technologies - Weâ€™re pioneers in JS, Node.js,
                React and Design Systems
              </Item>
              <Item>Grow a design & engineering culture</Item>
              <Item>
                Build custom software solutions that become your asset
              </Item>
              <Item>
                Deliver the right user experience at the right time, on the
                right channel
              </Item>
            </CheckList>
          </Col>
        </Row>
      </Grid>
    </PageSection>
  </Fragment>
);

export default WeEnable;
