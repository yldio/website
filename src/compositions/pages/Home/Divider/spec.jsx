import React from 'react';
import { shallow } from 'enzyme';
import WeEnable from './index';

test('compositions/homepage/WeEnable snapshot', () => {
  const wrapper = shallow(<WeEnable />);

  expect(wrapper).toMatchSnapshot();
});
