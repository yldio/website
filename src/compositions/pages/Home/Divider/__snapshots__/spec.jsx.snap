// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/homepage/WeEnable snapshot 1`] = `
<PageSection
  data-selector="home:we-enable"
  gradient={false}
  gradientHeight="50%"
>
  <PageSectionHeader
    description={null}
    title="We enable you to..."
  />
  <CheckList>
    <Item>
      Build custom software solutions that become an asset
    </Item>
    <Item>
      Deliver the right user experience at the right time, on the right channel
    </Item>
    <Item>
      Transform, innovate and differentiate at speed
    </Item>
    <Item>
      Grow capability inside your organisation - consulting, training, pairing and education
    </Item>
    <Item>
      Adopt cutting edge technologies - We're pioneers in Node.js, React and Design Systems
    </Item>
    <Item>
      Grow a technology culture
    </Item>
  </CheckList>
  <PageRule />
</PageSection>
`;
